reco_utils/recommender/fastai/fastai_utils.py

```diff
@@ -0,0 +1,84 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+
+import numpy as np
+import pandas as pd
+import fastai
+from fastprogress import force_console_behavior
+import fastprogress
+
+from reco_utils.common import constants as cc
+
+
+def cartesian_product(*arrays):
+    """Compute the cartesian product in fastai algo. This is a helper function.
+
+    Args:
+        arrays (tuple of np.array): Input arrays
+
+    Returns:
+        np.array: product
+
+    """
+    la = len(arrays)
+    dtype = np.result_type(*arrays)
+    arr = np.empty([len(a) for a in arrays] + [la], dtype=dtype)
+    for i, a in enumerate(np.ix_(*arrays)):
+        arr[..., i] = a
+    return arr.reshape(-1, la)
+
+
+def score(
+    learner,
+    test_df,
+    user_col=cc.DEFAULT_USER_COL,
+    item_col=cc.DEFAULT_ITEM_COL,
+    prediction_col=cc.DEFAULT_PREDICTION_COL,
+    top_k=0,
```

Comment:
Maybe None would be more meaningful than 0 as default value.

Reply:
fixed